src/Diagnostics.DataProviders/DataProviders/MdmDataProvider.cs

```diff
@@ -186,15 +188,13 @@ public async Task<IEnumerable<DataTable>> GetMultipleTimeSeriesAsync(DateTime st
                 foreach (var s in samplingType)
                 {
                     var table = new DataTable();
-                    table.Columns.Add("Metric", typeof(string));
                     table.Columns.Add("TimeStamp", typeof(DateTime));
-                    table.Columns.Add(s.ToString());
+                    table.Columns.Add("Metric", typeof(string));
+                    table.Columns.Add(s.ToString(), typeof(double));
```

Comment:
Instead of `s.ToString()` shouldn't this be parsed as Double

Reply:
No. The first parameter is the column name so it should be should. The second parameter is the type of the column.